subsys/bluetooth/audio/ascs.c

```diff
@@ -253,6 +253,20 @@ static void ascs_iso_recv(struct bt_iso_chan *chan,
 	}
 }
 
+static void ascs_iso_sent(struct bt_iso_chan *chan)
+{
+	struct bt_audio_ep *ep = CONTAINER_OF(chan, struct bt_audio_ep, iso);
+	struct bt_audio_stream_ops *ops = ep->stream->ops;
+
+	BT_DBG("stream %p ep %p", chan, ep);
+
+	if (ops != NULL && ops->sent != NULL) {
+		ops->sent(ep->stream);
+	} else {
+		BT_WARN("No callback for recv set");
```

Comment:
"No callback for *sent* set".

BTW, Is this warning needed?

Reply:
I actually think that the `sent` callback does not need this warning, as it is not needed to set this callback at all.